src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -18,7 +18,9 @@
     @Override
     public CommandResult executePrimitive(Model model, CommandHistory history) {
         requireNonNull(model);
-        model.resetData(new TaskManager());
+        TaskManager clearedTaskManager = new TaskManager();
+        clearedTaskManager.setAchievements(model.getAchievementRecord());
+        model.resetData(clearedTaskManager);
```

Comment:
add a comment explaining that you're not resetting all data.  Also, clearedTaskManager is a misnomer, it could be ambiguously implying that all data including achievements are reset and that's an intended effect